demo/app/messaging-view-model.ts

```diff
@@ -0,0 +1,165 @@
+import { alert } from "tns-core-modules/ui/dialogs";
+import { isIOS } from "tns-core-modules/platform";
+import { ios as iOSUtils } from "tns-core-modules/utils/utils";
+import * as firebase from "nativescript-plugin-firebase";
+import { IOSPushSettings, IosInteractiveNotificationActionOptions, IosInteractivePushSettings, PushNotificationModel } from "nativescript-plugin-firebase/messaging/messaging";
```

Comment:
Perhaps it's nicer to use the same casing for all these objects: `IOS..` vs `Ios..` looks a bit sloppy 😉